src/LinuxTracing/ThreadStateManager.h

```diff
@@ -86,6 +91,8 @@ class ThreadStateManager {
     // to the runnable state.
     pid_t wakeup_tid;
     pid_t wakeup_pid;
+
+    bool wait_for_callstack;
```

Comment:
And finally this should probably be called `has_wakeup_or_switch_out_callstack` (same for the constructor parameters).

Reply:
Done.